app/src/requirements.txt

```
@@ -5,5 +5,6 @@ geopy==1.21.0
 requests==2.23.0
 dill==0.3.1.1
 ipython==7.13.0
+xlrd==1.1.0
```

Seems that you are using `pandas.read_excel` and not `xlrd` anymore.
```suggestion
```

---

Ok sorry, it is a required dependency for the read_excel function ;)

```suggestion
xlrd==1.1.0  # Pandas dependency to read excel files
```